package.json

```diff
@@ -122,8 +122,8 @@
     "react-test-renderer": "^16.12.0",
     "rimraf": "^3.0.2",
     "semantic-release": "^17.4.0",
+    "styled-components": "^5.3.0",
```

Comment:
Why 2 different versions of `styled-components` ?

Reply:
Oh good point! I'll fix it

Reply:
@mkrds fixed 👍